src/prommis/uky/costing/custom_costing_example.py

```diff
@@ -0,0 +1,171 @@
+#####################################################################################################
+# “PrOMMiS” was produced under the DOE Process Optimization and Modeling for Minerals Sustainability
+# (“PrOMMiS”) initiative, and is copyright (c) 2023-2024 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory, et al. All rights reserved.
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license information.
+#####################################################################################################
+"""
+REE costing library
+This method leverages NETL costing capabilities.
+
+Other methods:
+
+    - get_fixed_OM_costs() to cost fixed O&M costs
+    - get_variable_OM_costs() to cost variable O&M costs
+    - costing_initialization() to initialize costing blocks
+    - display_total_plant_costs() to display total plant cost (TPC)
+    - display_bare_erected_costs() to display BEC costs
+    - get_total_BEC() to display the total BEC of the entire flowsheet
+    - display_flowsheet_cost() to display flowsheet cost
+    - calculate_REE_costing_bounds() to provide an estimate of costing bounds
+"""
+# TODO: Missing docstrings
+# pylint: disable=missing-class-docstring
+# pylint: disable=missing-function-docstring
```

Comment:
If these are not needed, should they just be removed?

Reply:
I'll try removing these too.